src/components/organisms/Badges/Badges.tsx

```diff
@@ -85,6 +87,22 @@ export const Badges: React.FC<{
       .filter((b) => b !== undefined);
   }, [relevantVisits, venues, currentVenue, relatedVenueIds]);
 
+  const badgeList = useMemo(
+    () =>
+      badges.filter(notEmpty).map((b) => (
+        <li className="badge-list-item" key={b.label}>
+          <Link to={getLocationLink(b.venue, b.room)}>
```

Comment:
[NB] Will this have impacts on counting, since presumably we're linking directly to a room? (as a wider thing.. where possible it would be good if we could capture counting when a page/thing loads (for on-platform stuff), rather than relying on us having a perfect implementation at every 'calling site'.

Reply:
@denisdimitrov FYI: if you get to looking at counting things, this may be another pattern of the types of things we're doing where we may not be correctly registering the count. I haven't deeply validated that, so treat this as a potential thing to look into if you get to it, rather than a "this is definitely related"